app/views/dashboard/index.html.erb

```diff
@@ -1,86 +1,230 @@
 <% if current_user %>
-  <table class="table table-sm table-borderless table-responsive navbar-brown align-top">
-    <tr>
-      <td align="center">
-        <div id='profile'>
-          <img id="profile_pic" src="<%= current_user.google_image_url %>" style="border-radius:50px; width:200px; height:auto;">
-          <br>
-          <br>
-          <%= link_to 'View Profile', profile_path(current_user.id), method: :get, class: "btn btn-sm btn-info navbar-custom btn-outline-dark" %>
-        </div>
+  <div class="container flex-wrap-reverse gray full-radius padding mb-3" id='profile-header'>
+    <div class="row item-center center-it-all">
+      <div class="col-lg">
+      </div>
+
+      <div class="col-lg">
+        <img id="profile_pic" src="<%= current_user.google_image_url %>" style="width:160px; height:auto;" class='round'>
+
+        <h1><%= current_user.full_name %></h1>
+
+        <%= link_to 'View Profile', profile_path(current_user.id), method: :get, class: "btn btn-sm btn-info button-bg btn-outline-dark" %>
+        <%= link_to 'Edit Profile', edit_profile_path(current_user.id), method: :get, class: "btn btn-sm btn-info button-bg btn-outline-dark" %>
+      </div>
+
+      <div class="col-lg fit-content">
+        <ul class='no-bullets'>
+          <li>
+            <%= embedded_svg('icons/basic_geolocalize-01.svg', class: 'icon') %>
+            &nbsp;<%= current_user.zip_code %>
+          </li>
```

Comment:
FYI ... this `embedded_svg` method is the method referenced in the `ApplicationHelper` module for displaying `.svg` image files.

Here is the reference again: https://blog.cloud66.com/using-svgs-in-a-rails-stack/

Reply:
Very fun to see how this is utilized in the views. Thank you!